### PR TITLE
task3

### PR DESCRIPTION
**Pull request by sonic-interlock**
What was done?
Two services and FE integration. Additional scope: async/awaits are used, not found path is covered

Link to product-service API
https://og1r0j6mg2.execute-api.eu-west-1.amazonaws.com/dev/products

LInk to FE MR 
https://d1xz8rgl6jxoup.cloudfront.net
https://github.com/sonic-interlock/nodejs-aws-fe/pull/2

In case SWAGGER file is not provided - please provide product schema in PR description
```
Cat {id: string; title: string; description: string; price: number}
```

### CODE REVIEW

**Comment by viacheslav-karnaukh** [comment]
I cannot see the usage of `await` in your codebase. Wasn't it done?

**Comment by sonic-interlock** [comment]
@viacheslav-karnaukh please take a look at handlers.js lines 11 and 20

File: product-service/handler.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+const data = require('./cats');
+
+const headers = {
+  'Content-Type': 'application/json',
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept',
+  'Access-Control-Allow-Methods': 'POST, GET, OPTIONS, PATCH, PUT',
+}
+
+module.exports.getProductsList = async event => {
```
Comment (balasphilip) [inline product-service/handler.js:11]: what is the purpose of using `async` keyword? Thera are no async operations in handler's body

Comment (sonic-interlock) [reply product-service/handler.js:11]: A cheap way not to create a Promise explicitly

**Pull request merged**